GUI/Controls/ManageMods.cs

```diff
@@ -1241,6 +1250,11 @@ private void ApplyHeaderGlyphs()
             }
             for (int i = 0; i < sortColumns.Count; ++i)
             {
+                if (ModGrid.Columns[sortColumns[i]] == null)
```

Comment:
Could we check `Contains(sortColumns[i])` instead? Usually trying to access a missing element in a C# collection throws an exception, so I would rather not trust that `DataGridViewColumnCollection.Item[]` is going to be gracious and forgiving here.